File: SEMANA_5/FA-16185/SE5-C/ejer2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SE5_C
{
    internal class ejer2
    {
        static void Main(string[] args)
        {
            int lado1, lado2, lado3;

            Console.WriteLine("Ingrese lado 1: ");
            lado1 = int.Parse(Console.ReadLine());
            Console.WriteLine("Ingrese lado 2: ");
            lado2 = int.Parse(Console.ReadLine());
            Console.WriteLine("Ingrese lado 3: ");
            lado3 = int.Parse(Console.ReadLine());
            Console.WriteLine();
            
            if (lado1 == lado2 && lado2 == lado3)
            {
                Console.WriteLine("EQUILATERO");
            }
            else if (lado1 == lado2 || lado2 == lado3 || lado3 == lado1)
            {
                    Console.WriteLine("ISOSCELES");
            }
            else
            {
                    Console.WriteLine("ESCALENO");
            }
            

        }
    }
}

File: SEMANA_5/FA-16185/SE5-PY/ejer2.py
lado1 = int(input("Ingrese lado 1:"))
lado2 = int(input("Ingrese lado 2:"))
lado3 = int(input("Ingrese lado 3:"))

if lado1 == lado2 and lado2 == lado3:
    print("EQUILATERO")
elif lado1 == lado2 or lado2==lado3 or lado3 == lado1:
    print("ISOSCELES")
else:
    print("ESCALENOS")


